internal-api/src/main/kotlin/com/ajaxproject/api/internal/warehousesvc/KafkaTopic.kt

```diff
@@ -0,0 +1,13 @@
+package com.ajaxproject.api.internal.warehousesvc
+
+import com.ajaxproject.api.internal.warehousesvc.NatsSubject.WAREHOUSE_PREFIX
```

Comment:
It's better to have independent prefixes for NATS and Kafka.

It's ok to have two constants with the same value.

Reply:
Done